test/payjp_test.dart

```diff
@@ -14,6 +14,24 @@ import 'package:payjp_flutter/payjp_flutter.dart';
 void main() {
   TestWidgetsFlutterBinding.ensureInitialized();
   final log = <MethodCall>[];
+  const tokenId = "tok_xxx";
+  const tokenJson = {
+    'id': tokenId,
+    'livemode': true,
+    'used': false,
+    'created': 0,
+    'object': 'token',
+    'card': {
+      'id': 'car_123',
+      'brand': 'visa',
```

Comment:
正しくは `Visa` なので、シリアライズ先で空とかになってそうですね。
```suggestion
      'brand': 'Visa',
```
特に影響なさそうですが一応